packages/polaris-viz/src/components/FunnelChartNext/components/FunnelTooltip/FunnelTooltip.scss

```diff
@@ -0,0 +1,10 @@
+.Tooltip {
+  top: 0;
+  left: 0;
+  position: absolute;
+  pointer-events: none;
+  // Matches --p-z-index-12
+  // https://polaris.shopify.com/tokens/z-index
+  z-index: 520;
+  max-width: 70%;
```

Comment:
I wonder if this might be less fragile as a px value. What does this end up being 70% of?

Reply:
Updated to px 👍